src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/brgemm_cpu_blocking.cpp

```diff
@@ -88,47 +63,32 @@ bool BrgemmCPUBlocking::mark_blocking_loops(LinearIR& linear_ir,
     const auto brgemm = ov::as_type_ptr<ov::intel_cpu::BrgemmCPU>(brgemm_expr->get_node());
     const auto type = brgemm->get_type();
 
-    if (stand_alone(type))
-        return ov::snippets::lowered::pass::BrgemmBlockingBase::mark_blocking_loops(linear_ir, brgemm_it, m_block, n_block, k_block);
+    auto res = ov::snippets::lowered::pass::BrgemmBlockingBase::mark_blocking_loops(linear_ir, brgemm_it, m_block, n_block, k_block);
 
-    brgemm_expr->get_input_port_descriptor(0)->set_subtensor({m_block, k_block});
-    brgemm_expr->get_input_port_descriptor(1)->set_subtensor({k_block, n_block});
-    brgemm_expr->get_output_port_descriptor(0)->set_subtensor({m_block, n_block});
+    if (stand_alone(type))
+        return res;
 
     const auto copy_b_expr = linear_ir.get_expr_by_node(brgemm->get_brgemm_copy());
-    copy_b_expr->get_input_port_descriptor(0)->set_subtensor({k_block, n_block});
-    copy_b_expr->get_output_port_descriptor(0)->set_subtensor({k_block, n_block});
+    copy_b_expr->get_input_port_descriptor(0)->set_subtensor({get_full_dim_value(), get_full_dim_value()});
```

Comment:
It's a fair question, but I don't think so because in order to repack the first line (or block) we need to access the whole `vnni_factor * N` subtensor. It means that to produce repacked K_blk x N_blk subtensor for brgemm, we would need to access `div_up(K_blk/vnni_factor) x N` subtensor. So basically repacking inside blocking cycles requires higher memory bandwidth than matrix multiplication.
BTW did me measure performance when the low precision blocking was initially introduced in [PR 23292](https://github.com/openvinotoolkit/openvino/pull/23292). @v-Golubev, maybe you have some data?